public/locales/en/app.ftl

```diff
@@ -0,0 +1,304 @@
+-product-name = Firefox Monitor
+-brand-name = Firefox
+-brand-Quantum = Firefox Quantum
+-brand-Mozilla = Mozilla
+-brand-HIBP = HIBP
+
+layout-Firefox = {-brand-name}
+# noun
+layout-support = Support
+layout-feedback = Give Feedback
+layout-terms = Terms and Privacy
+
+error-not-subscribed = This email address is not subscribed to {-product-name}.
+error-hibp-throttled = Too many connections to HIBP.
+error-hibp-connect = Error connecting to HIBP.
+error-hibp-load-breaches = Could not load breaches.
+
+hibp-notify-email-subject = {-product-name} Alert : Your account was involved in a breach.
+
+home-title = {-product-name}
+home-not-found = Page not found.
+
+oauth-invalid-session = Invalid session
+oauth-confirmed-title = {-product-name} : Subscribed
+
+scan-title = {-product-name} : Scan Results
+
+user-add-invalid-email = Invalid Email
+user-add-email-verify-subject = Verify your subscription to {-product-name}.
+user-add-title = {-product-name} : Confirm Email
+
+user-verify-token-error = Verification token is required.
+user-verify-email-report-subject = Your {-product-name} report
+user-verify-title = {-product-name} : Subscribed
+
+user-unsubscribe-token-error = Unsubscribe requires a token.
+user-unsubscribe-token-email-error = Unsubscribe requires a token and emailHash.
+user-unsubscribe-title = {-product-name} : Unsubscribe
+user-unsubscribe-survey-title = {-product-name} : Unsubscribe Survey
+user-unsubscribed-title {-product-name} : Unsubscribed
+
+## Password Tips
+
+pwt-section-headline = Passwords = Protection
+pwt-section-subhead = Your Private Information Is Only as Safe as Your Passwords
+pwt-section-blurb =  Your passwords protect more than your accounts, they protect every bit of personal information that resides in them. And hackers rely on bad habits, like using the same password everywhere or using common phrases (p@ssw0rd, anyone?), so that if they hack one account, they can hack many. Here are six ways to protect your accounts.
+
+pwt-headline-1 = Use a Different Password<br />for Every Account
+pwt-headline-2 = Create Strong<br />Passwords
+pwt-headline-3 = Treat Security Questions<br>Like Extra Passwords
```

Comment:
Would be nice to be consistent: `<br>` vs `<br />`

Reply:
I'm thinking these line breaks need to go since there isn't a way to know exactly where the line breaks need to be placed in each translation.